devel/python/python/ert/enkf/local_dataset.py

```diff
@@ -0,0 +1,57 @@
+from ert.cwrap import BaseCClass, CWrapper
+from ert.enkf import ENKF_LIB
+from ert.ecl import EclRegion
+
+class LocalDataset(BaseCClass):
+
+    def __init__(self, name):
```

Comment:
Please implement the `__contains__` method (requires a C function as well), so that we can check if a certain key is valid.